Crash when parsing πταιρω entry

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.benlinskey.greekreference/com.benlinskey.greekreference.views.detail.lexicon.LexiconDetailActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'android.text.SpannedString com.benlinskey.greekreference.data.lexicon.LexiconEntry.toSpanned(float)' on a null object reference
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2325)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2387)
at android.app.ActivityThread.access$800(ActivityThread.java:151)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1303)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5254)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.text.SpannedString com.benlinskey.greekreference.data.lexicon.LexiconEntry.toSpanned(float)' on a null object reference
at com.benlinskey.greekreference.views.detail.lexicon.LexiconDetailFragment.onCreateView(LexiconDetailFragment.java:103)
at android.app.Fragment.performCreateView(Fragment.java:2053)
at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:894)
at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
at android.app.BackStackRecord.run(BackStackRecord.java:834)
at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1452)
at android.app.Activity.performStart(Activity.java:6005)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2288)
... 10 more
```

--- a/GreekReference/src/main/java/com/benlinskey/greekreference/data/lexicon/LexiconXmlParser.java
+++ b/GreekReference/src/main/java/com/benlinskey/greekreference/data/lexicon/LexiconXmlParser.java
@@ -21,6 +21,7 @@ import android.text.Spannable;
 import android.text.SpannableString;
 import android.text.TextUtils;
 import android.text.style.StyleSpan;
+import android.util.Log;
 import android.util.Xml;
 
 import org.xmlpull.v1.XmlPullParser;
@@ -52,6 +53,11 @@ public class LexiconXmlParser {
             parser.setFeature(XmlPullParser.FEATURE_PROCESS_DOCDECL, true);
             parser.setInput(in, null);
             return readXml(parser);
+        } catch (Exception e) {
+            // TODO: Fail gracefully here -- allow the user to submit a bug report.
+            // See issue #141.
+            Log.e(this.getClass().getCanonicalName(), "Parsing error", e);
+            return null;
         } finally {
             in.close();
         }
@@ -123,11 +129,17 @@ public class LexiconXmlParser {
     private void readForm(XmlPullParser parser, LexiconEntry entry) throws XmlPullParserException,
             IOException {
         while (parser.next() != XmlPullParser.END_TAG) {
-            String name = parser.getName();
-            if (name.equals("orth")) {
-                entry.setOrth(readText(parser));
+            if (parser.getEventType() == XmlPullParser.TEXT) {
+                String text = parser.getText();
+                String orth = entry.getOrth();
+                entry.setOrth(orth + text);
             } else {
-                skip(parser);
+                String name = parser.getName();
+                if (name.equals("orth")) {
+                    entry.setOrth(readText(parser));
+                } else {
+                    skip(parser);
+                }
             }
         }
     }
@@ -309,4 +321,5 @@ public class LexiconXmlParser {
 
         return result;
     }
+
 }

